docs/detections/rules-ui-create.asciidoc

```diff
@@ -138,28 +137,31 @@ then:
 .. Use the filter and query fields to create the criteria used for detecting
 alerts.
 +
-NOTE: You can use {kib} saved queries (image:images/saved-query-menu.png[Saved query menu,17,17]) and queries from saved Timelines (*Import query from saved Timeline*) as rule conditions.
-+
-For example, the following rule detects when the `vssadmin delete shadows`
+The following example (based on the prebuilt rule <<prebuilt-rule-0-14-2-volume-shadow-copy-deleted-or-resized-via-vssadmin, Volume Shadow Copy Deleted or Resized via VssAdmin>>) detects when the `vssadmin delete shadows`
 Windows command is executed:
 
 ** *Index patterns*: `winlogbeat-*`
 +
-> Winlogbeat ships Windows event logs to {es-sec}.
+Winlogbeat ships Windows event logs to {elastic-sec}.
 
 ** *Custom query*: `event.action:"Process Create (rule: ProcessCreate)" and process.name:"vssadmin.exe" and process.args:("delete" and "shadows")`
 +
-> Searches the `winlogbeat-*` indices for `vssadmin.exe` executions with
+Searches the `winlogbeat-*` indices for `vssadmin.exe` executions with
 the `delete` and `shadow` arguments, which are used to delete a volume's shadow
 copies.
 +
 [role="screenshot"]
-image::images/rule-query-example.png[]
-+
-TIP: This example is based on the
-<<prebuilt-rule-0-14-2-volume-shadow-copy-deleted-or-resized-via-vssadmin, Volume Shadow Copy Deleted or Resized via VssAdmin>> prebuilt rule.
+image::images/rule-query-example.png[Rule query example]
+
+.. You can use {kib} saved queries (image:images/saved-query-menu.png[Saved query menu,18,18]) and queries from saved Timelines (*Import query from saved Timeline*) as rule conditions.
 +
-. Continue with <<preview-rules, previewing the rule>> (optional) or click **Continue** to <<rule-ui-basic-params, configure basic rule settings>>.
+When you use a saved query, the *Load saved query "_query name_" dynamically on each rule execution* check box appears:
```

Comment:
is this worth to mention changes on rule details page for saved query rules?

https://github.com/elastic/kibana/issues/139250#issuecomment-1247918828

Reply:
Hi @vitaliidm I think it's worth mentioning, but maybe not in this section. The reader is in the middle of a kind of long procedure of creating a rule, so they won't be able to observe the details page until they're done with all the steps and save the rule.

Currently we actually don't have a dedicated docs page for the rule details page, but we're planning to create one for reasons just like this. I've added a [comment](https://github.com/elastic/security-docs/issues/2560#issuecomment-1278324186) to that docs issue so we can be sure to include this detail.